pkg/storage/tsdb/config.go

```diff
@@ -138,6 +138,9 @@ type TSDBConfig struct {
 
 	// How often to check for idle TSDBs for closing. DefaultCloseIdleTSDBInterval is not suitable for testing, so tests can override.
 	CloseIdleTSDBInterval time.Duration `yaml:"-"`
+
+	// Positive value enables experiemental support for exemplars. 0 or less to disable.
+	MaxExemplars int `yaml:"-"`
```

Comment:
Expose it in YAML config too, please. Then run `make doc`.

Reply:
Not sure `make doc` is doing what it should here, it added entries to `docs/block-storage/querier.md`, `docs/blocks-storage/store-gateway.md` in addition to `/docs/configuration/config-file-reference.md`. Is that right @pracucci ?

Reply:
It's right because the whole blocks storage config is currently displayed in all the pages you referenced. It's something we can further improve, but don't worry about it in this PR.